kfs-core/src/main/resources/edu/arizona/kfs/pdp/ojb-pdp.xml

```diff
@@ -119,4 +119,89 @@
+        <orderby name="invoiceNumber" sort="ASC" />
+        <inverse-foreignkey field-ref="loadDate" />
+        <inverse-foreignkey field-ref="fileName" />
+    </collection-descriptor>    
+  </class-descriptor>
+
+  <class-descriptor class="edu.arizona.kfs.pdp.businessobject.ShippingInvoiceHistory" table="PDP_SHIPPING_INVOICE_HIST_T">
+    <field-descriptor name="loadDate" column="LOAD_DT" jdbc-type="TIMESTAMP" primarykey="true"/>
+    <field-descriptor name="fileName" column="FILE_NM" jdbc-type="VARCHAR" primarykey="true"/>
+    <field-descriptor name="invoiceNumber" column="INVOICE_NBR" jdbc-type="VARCHAR" primarykey="true"/>
+    <field-descriptor name="openCustomField" column="OPEN_CUSTOM_FIELD" jdbc-type="VARCHAR"/>
+    <field-descriptor name="shippingCompany" column="SHIPPING_CO" jdbc-type="VARCHAR"/>
+    <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true"/>
+    <field-descriptor name="versionNumber" column="VER_NBR" jdbc-type="BIGINT" locking="true"/>
+    <field-descriptor name="creationDate" column="CREATION_DT" jdbc-type="DATE"/>
+    <field-descriptor name="transactionRefNumber" column="TRANSACTION_REF_NBR" jdbc-type="VARCHAR"/>
+    <field-descriptor name="invoiceDate" column="INVOICE_DT" jdbc-type="DATE"/>
+    <field-descriptor name="totalInvoiceCharge" column="TOTAL_INVOICE_CHARGE" jdbc-type="DECIMAL" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbKualiDecimalFieldConversion"/>
+    <field-descriptor name="billToAccountNumber" column="BILL_TO_ACCT_NBR" jdbc-type="VARCHAR"/>
+    <field-descriptor name="invoiceType" column="INVOICE_TYP_CD" jdbc-type="VARCHAR"/>
+    <field-descriptor name="settlementOption" column="SETTLEMENT_OPTION_CD" jdbc-type="VARCHAR"/>
+    <field-descriptor name="totalInvoiceTransactions" column="TOTAL_INVOICE_TRANS_NUM" jdbc-type="BIGINT" conversion="org.kuali.rice.core.framework.persistence.ojb.conversion.OjbKualiIntegerFieldConversion"/>
+    <field-descriptor name="otherAccountNumber" column="OTHER_ACCT_NBR" jdbc-type="VARCHAR"/>
+    
+    <collection-descriptor name="invoiceTrackingElements" element-class-ref="edu.arizona.kfs.pdp.businessobject.ShippingInvoiceTrackingHistory" collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" auto-retrieve="true" auto-update="object" auto-delete="object" proxy="true">
+        <orderby name="sequenceRowNumber" sort="ASC" />
+        <inverse-foreignkey field-ref="loadDate" />
+        <inverse-foreignkey field-ref="fileName" />
+        <inverse-foreignkey field-ref="invoiceNumber" />
+    </collection-descriptor>    
+  </class-descriptor>
+
+  <class-descriptor class="edu.arizona.kfs.pdp.businessobject.ShippingInvoiceTrackingHistory" table="PDP_SHIPPING_INV_TRACK_HIST_T">
+    <field-descriptor name="loadDate" column="LOAD_DT" jdbc-type="TIMESTAMP" primarykey="true"/>
```

Comment:
Missing REBILL_IND and SHIP_DT columns from the table, not sure if it's intentional.